src/components/figures/chart/chartJs/chartjs.ts

```diff
@@ -6,6 +6,7 @@ import { SpreadsheetChildEnv, UID } from "../../../../types";
 import { ChartJSRuntime } from "../../../../types/chart/chart";
 import { chartJsExtensionRegistry, registerChartJSExtensions } from "./chart_js_extension";
 import { ChartAnimationStore } from "./chartjs_animation_store";
+import { chartColorScalePlugin } from "./chartjs_colorscale_plugin";
```

Comment:
I would add at least some demo data, I picked up the task randomly and ... wel don't know what kind of data I should put in it